models/DataObject/Fieldcollection.php

```diff
@@ -109,10 +109,16 @@ public function setFieldname($fieldname)
     public function getItemDefinitions()
     {
         $definitions = [];
-        foreach ($this->getItems() as $item) {
-            $definitions[$item->getType()] = $item->getDefinition();
+        $items = $this->getItems();
+        foreach ($items as $key => $item) {
+            if($item == null) {
```

Comment:
Preferably add a space between 'if' and '(' and use exact matching `===` (`"" == null` results in `true`).

Reply:
@kjkooistra-youwe Thanks, I have adopted the changes you requested.